Gemfile.lock

```diff
@@ -172,9 +174,12 @@ DEPENDENCIES
   jekyll-sitemap
   jemoji
   kramdown
+  kramdown-parser-gfm
   liquid
   listen
 
 RUBY VERSION
    ruby 2.5.5p157
 
+BUNDLED WITH
```

Comment:
please remove these two lines, they only cause problems when preparing updates ona different system than the target one

Reply:
Done